automated install/basic-install.sh

```diff
@@ -2301,6 +2301,11 @@ copy_to_install_log() {
 }
 
 disableLighttpd() {
+    # Return early when lighttpd is not installed
+    if [[ ! -f /etc/lighttpd/lighttpd.conf ]]; then
```

Comment:
We are not touching any lighttpd config file when disabling/stopping it. I think we should instead use something like `check_service_active()` (which we already have) and/or 

```
nanopi@nanopi:~$ systemctl is-enabled lighttpd.service 
lighttpd.service is not a native service, redirecting to systemd-sysv-install.
Executing: /lib/systemd/systemd-sysv-install is-enabled lighttpd
enabled
nanopi@nanopi:~$ systemctl is-active lighttpd.service 
inactive

```

Reply:
The routine calling this is only run once during 5 -> 6 migration, this code you are commenting on here will only be running once and my check was for if `lighttpd` is installed at all. I'm not really seeing how the config file being there may not be a good sign of it being installed. I don't think anyone would have installed and then removed it again leaving the config in place. I guess a Pi-hole repair/upgrade would have then reinstalled it.

Anyway, `check_service_active` is fine for me, too. I will push another commit